pages/linux/makepkg.md

```diff
@@ -1,22 +1,22 @@
 # makepkg
 
-> Creates a package installable with the `pacman` package manager.
-> Runs the commands from a `PKGBUILD` file to build the package.
+> Creates a package which can be used with `pacman`.
+> Uses the `PKGBUILD` file in the current working directory.
```

Comment:
Should we mention `current working dir` here ?
It does take an option `-p` where we can supply any `PKGBUILD` file.
How about:
```suggestion
> Uses the `PKGBUILD` file in the current working directory by default.
```